content/concepts/appendix/glossary.md

```diff
@@ -61,7 +61,7 @@ For example: `/ip4/192.0.2.0/udp/1234` encodes two protocols along with their es
 
 Multiaddresses can be composed to describe multiple "layers" of addresses.
 
-For more detail, see [Concepts > Addressing](/concepts/addressing/), or the [multiaddr spec](https://github.com/multiformats/multiaddr), which has links to many implementations.
+For more detail, see [Addressing](/concepts/fundamentals/addressing.md), or the [multiaddr spec](https://github.com/multiformats/multiaddr), which has links to many implementations.
```

Comment:
```suggestion
For more detail, see [Addressing](/concepts/fundamentals/addressing.md), or the [multiaddr spec](https://github.com/multiformats/multiaddr), which has links to many implementations.
```
This breaks the link actually: https://bafybeialordnq273b2gwnvfbqc25prs75qu6z5gzlwom2febrnoyz3x5dm.on.fleek.co/concepts/appendix/glossary/

This occurs for other links in the glossary page. I don't think you should add `.md` to the path

Reply:
Woops, my mistake. I distinctly remembered Hugo being able to do that, but I was mistaken.

We could convert every .md link to use the [`ref`](https://gohugo.io/content-management/cross-references/) tag, but that would be even more verbose and also wouldn't work with lychee, so not really an option.

I will see what options are available for generating testable links.